it/api/endpoints/lossClaim/list/v3/ListLossClaimsControllerISpec.scala

```diff
@@ -148,17 +143,41 @@ class ListLossClaimsControllerISpec extends V3IntegrationBaseSpec {
     }
   }
 
+  private trait NonTysTest extends Test {
+    override def taxYear: Option[String] = Some("2019-20")
+
+    override def downstreamUrl: String = s"/income-tax/claims-for-relief/$nino"
+
+    override def queryParams: Seq[(String, String)] =
+      Seq("taxYearClaimedFor" -> taxYear, "typeOfLoss" -> typeOfLoss, "businessId" -> businessId, "typeOfClaim" -> claimType)
+        .collect {
+          case (k, Some(v)) => (k, v)
+        }
+  }
+
+  private trait TysIfsTest extends Test {
+    override def taxYear: Option[String] = Some("2023-24")
+
+    override def downstreamUrl: String = s"/income-tax/claims-for-relief/23-24/$nino"
+
+    override def queryParams: Seq[(String, String)] =
+      Seq("taxYearClaimedFor" -> taxYear, "typeOfLoss" -> typeOfLoss, "businessId" -> businessId, "typeOfClaim" -> claimType)
+        .collect {
+          case (k, Some(v)) => (k, v)
+        }
+  }
+
   "Calling the ListLossClaims endpoint" should {
 
     "return a 200 status code" when {
 
-      "query for everything" in new Test {
+      "query for everything" in new NonTysTest {
```

Comment:
Not a question about anything you've written, but looks like `query for everything` means providing no query params (as got Map.empty for the QPs)... so if we don't provide any of the QPs then we get all possible data types back?